src/AuthContext.tsx

```diff
@@ -27,6 +28,27 @@ export const hasCodeInUrl = (location: Location): boolean => {
   );
 };
 
+export const errorInUrl = (location: Location): Error | null => {
```

Comment:
This should probably not be a part of this pull request (as it is part of #552)

Reply:
Should i help rebasing this, such that it can land anytime soon?

Reply:
prepared in #582